GOVERNANCE.md

```diff
@@ -24,8 +24,6 @@ Changes to CC membership should be posted in the agenda, and may be suggested as
 
 No more than one-fourth of the CC members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CC member, or a change of employment by a CC member, creates a situation where more than one-fourth of the CC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CC members affiliated with the over-represented employer(s).
 
-The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
```

Comment:
I don't think removing this is needed. AFAICT the comm-comm itself is a community project and the 3 months observation should in itself be considered activity. So this clause is congruent with the current onboarding process.

https://github.com/nodejs/community-committee/blob/6dad5972ea9bfa17e8b69081cf167df3fa9f6139/GOVERNANCE.md#L89

Reply:
I see what you're thinking- but with the definition of "Member" you're in a chicken & egg problem. (You're not a Member, so therefor you can't become a Member)

Ultimately, I don't think it is a big deal to remove it.

Reply:
I thought this was more to specify that the CommComm is comprised of the Individual Membership Directors (who are members by being the Individual Membership Directors) and also the active members of the committee itself.

Reply:
@hackygolucky yes- I presume that was the case too. However, should absent IDs be considered part of CommComm? I don't think so. They certainly shouldn't be allowed to drop in once a year (for example) with voting power!! I feel they should just be CommComm members if they want to be and be held to the same requirements.

I noticed that I missed a PR from @bnb recently that listed me as a Member of CommComm via the ID role- which is a bit offensive since I was a founding member of CommComm and it precluded my ID role. So, after I'm done with my ID term- am I off the CommComm? Ouch! Maybe this wasn't the intent- but that's how it now seems to me.

The IDs can still coordinate with the Board with their needs without being a Member- and if the IDs are absent... then it would make a real case for CommComm to have it's own director.

If anything, perhaps we should write somewhere that _any_ Director is allowed to Observe a meeting.